File: OEM/Domain/ComplementaryTaskCategories/ComplementaryTaskCategoryDto.cs
using System;

namespace OEMAPI.Domain.ComplementaryTaskCategories
{
    public class ComplementaryTaskCategoryDto
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }
}

File: OEM/src/dto/ComplementaryTaskCategoryDTO.ts
export default interface ComplementaryTaskCategoryDTO {
  id: string;
  code: string;
  name: string;
  description: string;
}

export interface CreateComplementaryTaskCategoryDTO {
  code: string;
  name: string;
  description: string;
}

export interface UpdateComplementaryTaskCategoryDTO {
  code?: string;
  name?: string;
  description?: string;
}